run_dir/static/js/running_notes.js

```diff
@@ -147,20 +148,48 @@ function filter_running_notes(search){
         }
     });
 }
+//Count rn's of each category and add badge with number to filter dropdown
+function count_cards(){
+    var cat_cards = {};
+    var all = 0;
+    $('#running_notes_panels').find('.badge').each(function(){
+        var label = $.trim($(this).text());
+        all++
+        if (label){
+            if (label in cat_cards){
+                cat_cards[label]++;
+            }else{
+                cat_cards[label] = 1;
+            }
+        }
+    });
+    $('.btn_count').append('All'+'&nbsp;'+'<span class="badge bg-secondary">'+all+'</span>');
+    $('#rn_category').next().find('.dropdown-item').each(function(){
+        var label = $.trim($(this).text())
+        if (label == 'All'){
+            $('.all_count').append('&nbsp;'+'<span class="badge bg-secondary">'+all+'</span>');
+        }else if (label in cat_cards){
+            $(this).append('&nbsp;'+'<span class="badge bg-secondary">'+cat_cards[label]+'</span>');
+        }else{
+            $(this).prop('disabled', true);;
+        }
+    });
+}
+
 //Filter notes
 $('#rn_search').keyup(function() {
     var search=$('#rn_search').val();
     filter_running_notes(search);
 });
 
-$(document).ready(function() {
-    $('#rn_category').change(function() {
-        var search=$('#rn_category :selected').text();
-        if (search == 'All'){
-            search='';
-        }
-        filter_running_notes(search);
-    });
+//Filter dropdown
+$('.note_category').on('click', function(){
```

Comment:
If this is the only place that the `note_category` class is used, you can remove it from all of the `li` elements and just use one of the existing parent identifiers instead:

```suggestion
$('#rn_category ul li button').on('click', function(){
```

Reply:
I think it would work with `$('#rn_category ~ ul > li > button').on('click', function(){...`

Reply:
It works!